sdk/core/Azure.Core/src/TelemetryDetails.cs

```diff
@@ -80,16 +87,80 @@ internal static string GenerateUserAgentString(Assembly clientAssembly, string?
             {
                 version = version.Substring(0, hashSeparator);
             }
-            var platformInformation = $"({RuntimeInformation.FrameworkDescription}; {RuntimeInformation.OSDescription})";
+            runtimeInformation ??= new RuntimeInformationWrapper();
+            var platformInformation = EscapeProductInformation($"({runtimeInformation.FrameworkDescription}; {runtimeInformation.OSDescription})");
 
-            return applicationId != null
+            var ua = applicationId != null
```

Comment:
Just curious: why?

Reply:
Didn't mean to leave that in - thanks for catching!